Core/src/org/sleuthkit/autopsy/communications/AccountsNodeFactory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2011-2017 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.communications;
+
+import java.util.List;
+import org.openide.nodes.ChildFactory;
+import org.openide.nodes.Node;
+import org.sleuthkit.datamodel.Account;
+
+
+
+class AccountsNodeFactory extends ChildFactory<Account> {
+
```

Comment:
It looks like we are plowing a green field in terms of the view that will display the AccountNodes, am I right? If so, I still think we should go with lazy child node creation here.